packages/asc-ui/src/components/Blog/Article/Article.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import ArticleStyle from './ArticleStyle'
+import { Props } from '../BlogPost/BlogPost'
+
+const Article: React.FC<Props & React.HTMLAttributes<HTMLElement>> = ({
```

Comment:
Replace HTMLAttributes type, which is huge with a small custom type/interface containing just the className